viewing_party/party.py

```diff
@@ -1,23 +1,179 @@
 # ------------- WAVE 1 --------------------
 
+from re import I
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if title == None or genre == None or rating == None:

```

Comment:
Great use of this guard pattern here to return early from the function if those parameters are `None`! You could even define `new_movie` after this check because the empty dictionary doesn't need to be created if we're returning early.